port/cpl_conv.cpp

```diff
@@ -1863,6 +1935,9 @@ void CPL_STDCALL CPLSetConfigOption(const char *pszKey, const char *pszValue)
 
     g_papszConfigOptions = const_cast<volatile char **>(CSLSetNameValue(
         const_cast<char **>(g_papszConfigOptions), pszKey, pszValue));
+
+    NotifyOtherComponentsSetConfigOptiond(pszKey, pszValue,
```

Comment:
Does the ending `d` in the method name has a special meaning?

Reply:
just a copy& replace error. The intended function name is ``NotifyOtherComponentsConfigOptionChanged``. Fixed